.gitignore

```diff
@@ -1,2 +1,4 @@
+.idea
+
```

Comment:
```suggestion
```

Reply:
This kind of files [should be ignored at a global scope](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) 😊